ci-operator/config/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-master.yaml

```diff
@@ -37,14 +37,23 @@ tests:
 - as: verify
   commands: |
     go get -u golang.org/x/tools/imports
-    make verify -k
+    make verify -k --warn-undefined-variables
   container:
     from: bin
 
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
```

Comment:
why reassign home?

Reply:
this is set to a dir (I think `/`) which the user doesn't have permissions to, bad image

fyi @stevekuznetsov

Reply:
`$HOME` is unset ... why do you need it?